### PR TITLE
Move issues-scan to queue

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: tasks/webhook [hunk 1]
```diff
@@ -171,6 +173,19 @@ class ReleaseHandler(github_handler.GithubHandler):
         publish_to_queue('webhook', json.dumps(body))
         return None
 
+    def handle_issues_event(self, event, request):
+        action = request['action']
+        if event == 'issues' and action not in ['opened', 'synchronize', 'edited']:
```
Comment (martinpitt) [inline tasks/webhook]: What is "synchronize" for an issue?

Also, isn't the `event == 'issues'` redundant here?

File: tasks/webhook [hunk 2]
```diff
@@ -171,6 +173,19 @@ class ReleaseHandler(github_handler.GithubHandler):
...
+            logging.info("action not in ['opened', 'synchronize'], skipping issue")
```
Comment (martinpitt) [inline tasks/webhook]: not the same set; how about making this more useful for debugging and say something like `"action %s unknown, skipping event" % action`?

File: tasks/webhook [hunk 3]
```diff
@@ -146,7 +146,7 @@ class ReleaseHandler(github_handler.GithubHandler):
             return None
 
         body = {
-            'event': 'pull_request',
+            'event': event,
```
Comment (martinpitt) [inline tasks/webhook]: Good call, and then the whole "build body and publish it to webhook" can be factorized.

File: tasks/webhook (hunk 1 above)
Comment (Gundersanne) [reply tasks/webhook]: wel li want to catch someone unchecking the checkbox; i should probably just see what event that is and catch that :)

Comment (Gundersanne) [reply tasks/webhook]: just realized then if we want to listen to checkboxes, we'd also need to listen to PRs, but a different action.

Comment (Gundersanne) [reply tasks/webhook]: just verified, checkboxes on issues are 'edited'.  On PRs we'll need to listen to the same, but pass it to issue-scan I guess ?

Comment (martinpitt) [reply tasks/webhook]: Right, whenever the description of a PR changes we need to pipe it through issue-scan.

**Comment by Gundersanne** [comment]
reordered and squashed!

**Comment by Gundersanne** [comment]
@marusak This is the current deployment. Seems to be working fine. As you can see the checkouts are now done in XDG_CACHE_HOME

**Pull request merged**